tracer/test/snapshots/AwsLambdaTests.verified.txt

```diff
@@ -1978,5 +1957,143 @@
     Metrics: {
       _dd.top_level: 1.0
     }
+  },
+  {
+    TraceId: Id_19,
+    SpanId: Id_20,
+    Name: placeholder-operation,
+    Resource: placeholder-operation,
+    Service: placeholder-service,
```

Comment:
These placeholders definitely seem like a bug, I'm pretty certain you shouldn't have them